---
title: "manuscript"
output: word_document
date: "2023-12-11"
---

```{r setup, include=FALSE}
load(file.path("data", "v5_reference_mapped_all_filtered.RData"))
```

## Methods

### Single-cell sorting and analysiss
Tumor tissues were excised and mechanically dissociated in ice-cold HBSS supplemented with glucose (10 mM) and HEPES (10 mM) and using glass shearing with a 10-ml Potter-Elvehjem pestle and glass-tube homogenizer (Merck). The resulting suspension was passed through a 70-µm cell strainer (BD Biosciences). The cells were incubated with FC blocking (BD Biosciences). Simultaneously, 20 µl of Cell Multiplexing Oligo (CMO) lipid solution (3' CellPlex Kit, 10X Genomics) was added for multiplexing for 20 minutes. Afterwards, cells were washed with flow cytometry buffer and incubated for additional 20 minutes with anti-CD45 or anti-CD45.2 antibodies (Biolegend). Cell sorting was performed on a MoFlo Astrios (Beckman Coulter). A total of 50,000 cells were sorted per sample and analyzed using 10X Genomics 3’ single-cell mRNA sequencing (v3.1 chemistry), resulting in a final yield of approximately 10,000 sequenced cells per reaction. 

Libraries were prepared following the manufacturer’s instructions and sequenced on an Illumina NextSeq 550 machine. The resulting fastq files were demultiplexed and aligned to a mouse reference transcriptome (refdata-gex-mm10-2020-A provided on the 10X Genomics website) using the “cellranger multi” function within the CellRanger version 7.1.0 software. 

Single-cell data analysis was performed in `r R.version$version.string`. Cell doublets were identified and excluded using a combination of the scDblFinder version `r packageVersion('scDblFinder')` and SingleCellExperiment version `r packageVersion('SingleCellExperiment')` packages. Also, cell with more than 5 percent mitochondrial gene content and 5,000 or more detected genes were excluded. Downstream analysis of n=`r formatC(dim(all)[2], format="f", big.mark=",", digits=0)` cells was conducted in Seurat version `r packageVersion('Seurat')` [doi: 10.1038/s41587-023-01767-y]. Gene Expression modules were quantified using the UCell package version `r packageVersion("UCell")` [doi: 10.1016/j.csbj.2021.06.043]. Cluster abundance enrichment analysis was done using hypergeometric testing in R. The local abundance of  the treatment associated with each cell was visualized using the ggpointdensity package version `r packageVersion("ggpointdensity")`. Pseudotime and gene ontology analyses were performed using monocle3 version `r packageVersion('monocle3')` and ClusterProfiler version `r packageVersion('ClusterProfiler')` packages [doi: 10.1038/nbt.2859, 10.1016/j.xinn.2021.100141]. Significantly enriched genes in monocle were identified based on adjusted P values and Moran's I paremeters assessed with the graph_test() function of monocle3. For plotting, we utilized tidyverse version `r packageVersion('tidyverse')` and ComplexHeatmap version `r packageVersion('ComplexHeatmap')` [doi: 10.21105/joss.01686, 10.1093/bioinformatics/btw313].

## Results

### scRNA-seq identifies distinct immune-activating signatures for CAR T cells and NK cells and an immunosuppressive signature for CAR BMDM in the glioma microenvironment 

To investigate long-term effects of intratumoral CAR T cell, CAR NK cell and CAR BMDM implantation on the tumor microenvironment, we investigated the transcriptome of tumor infiltrating immune cells. To this end, we FACS-sorted CD45+ tumor infiltrating cells of GL-261 tumor–bearing mice 5 days after CAR immune cell injection and subjected them to droplet-based scRNA-seq (**Fig. 2A**). We assessed single-cell RNA profiles of at least n=`r min(table(all$group))` leukocytes per sample that passed the quality controls (**Suppl. Fig. _**). The resulting n=`r formatC(dim(all)[2], format="f", big.mark=",", digits=0)` cells were visualized using Uniform Manifold Approximation and Projection (UMAP).

We identified different myeloid compartments including monocytes, a large cluster of TAMs and microglia (**Fig. 2A**, **Suppl. Fig. _A**). Furthermore, we found granulocytes, dendritic cells (DC), NK, T , B and few mast cells. Of all CAR immune cells, only CAR T cells could still be identified as a distinct subset in cluster 12 (**Fig. 2B-C**). CAR T cell treatment was associated with a higher presence of TAMs (cluster 26) and granulocytes (clusters 1, 4) (**Fig. 2B-C**,**Suppl. Table _**). CAR NK cells were associated with a higher abundance of transitory monocytes (clusters 5, 11, 18, 2) and cDC2 (cluster 17), potentially indicative of higher myeloid recruitment. CAR BMDMs were associated with a higher abundance of transitory monocytes (cluster 2), TAMs (cluster 3) and NK cells (cluster 9) (**Fig. 2B-C**,**Suppl. Table _**). The control group showed a unique signature of enriched TAMs (clusters 0, 8, 10, 13) and granulocytes (clusters 1, 15, 25). Thus, different CAR cell types induced distinct polarizations of the tumor immune microenvironment.

Gene ontology enrichment analysis further corroborated these distinct effects on the different immune subsets. The T cell clusters were shifted towards a cytotoxic state by CAR T cells, while CAR NK cells and BMDMs consistently induced ontology term associated with anti-viral immune defense (**Fig. 3D**). Additionally, CAR BMDM cells induced genes associated with antigen presentation. The control groups displayed induction of leukocyte chemotaxis terms. The NK cell cluster showed attenuated responses to the different CAR cell treatments, potentially due to limited cell numbers (**Fig. 3E**). The myeloid clusters, including classical monocytes, transitory monocytes and TAMs, showed multifaceted changes following CAR immune cell treatment. Whereas the control group showed genes associated with leukocyte migration and type II interferon response, CAR T cell treatment induced a metabolic rewiring, characterized by glycolytic transcriptional programs (**Fig. 3F**). CAR NK cells shifted the myeloid compartment towards anti-viral immune response. CAR BMDM were characterized by non-cell-autonomous cell lysis transcriptional programs driven by NK cells. In sum, CAR T cell treatment was associated with a potent cytotoxic lymphocyte activity and a concomitant metabolic re-wiring of the myeloid compartment. CAR NK treatment potentiated anti-viral signaling. CAR BMDM appeared to drive anti-viral immune responses in a non-cell autonomous way via NK cells.

Next, we further dissected the different effects of CAR immune cell treatment on the myeloid compartment, representing the largest immune cell subset within the glioma tumor microenvironment. To this end, we conducted monocle3-based pseudotime analysis [doi: 10.1038/nbt.2859] with the classical monocyte cluster 20 as the earliest timepoint (**Fig. 3G**). On the identified transcriptional continuum, transitory monocytes were followed by TAMs, that in turn appeared to upregulate microglia-associated transcriptional programs (**Suppl. Fig. _B**). The upregulation of microglia-associated transcriptional programs may represent a combined underlying process: i. part of these cells is derived from microglia or ii. they are subject to immune imprinting by the brain tissue [doi: 10.1016/j.neuron.2018.05.014]. Analysis of significantly enriched genes within the six emerging gene modules showed that early "pseudo"-timepoints were associated with wound healing and anti-viral response terms (**Fig. 3H**). Intermediate timepoints were characterized by antigen processing-associated genes, transitioning to macrophage migration and lipid metabolism states at late pseudotime stages. Abundance analysis of the respective stages showed enrichment of control groups at the intermediate and late-stage associated gene expression modules 2, 5 and 6 (**Fig. 2I**). Likewise, CAR T cell treatment was associated with the intermediate and late stage-associated modules 2 and 4, while CAR NK treatment was significantly associated with the early and intermediate-stage modules 1, 3 and 6. CAR BMDM treatment showed an enrichment in the early-stage module 3 (along with CAR NK cells) and the late-stage module 4 (along with CAR T cells). In summary, CAR T cell treatment was associated with an metabolic rewiring of TAMs while CAR NK treatment increased the abundance of transitory monocytes. CAR BMDB treatment showed features of both.


## Figure legends
Figure 2. Single-cell immune cell profiling of the glioma immune microenvironment during different CAR immune treatments. (A) UMAP of n=`r formatC(dim(all)[2], format="f", big.mark=",", digits=0)` cells from different lineages, indicated in the plot. (B) Visualization of the local abundance of the different conditions. The color-coding represents the presence of cells of the same condition across the different neighborhoods within the UMAP latent space. (C) Differential enrichment testing of cells from the respective conditions based on hypergeometric testing with Benjamini-Hochberg adjustment for multiple testing. (D-F) Treatment-dependent enrichment of gene ontology terms across the indicated immune cell subsets. The dot size indicates the gene ratios of differentially expressed genes found in the present dataset versus genes contained in the respective gene ontology term. The color-coding represents the P value of the underlying enrichment test. (G) UMAP of the myeloid cell clusters color-coded for the pseudotime calculated using monocle3. The line represents a path of stepwise gene expression changes. (H) Visualization of the stepwise gene expression along the pseudotime trajectory with the representative gene ontology terms enriched in the respective gene expression modules along the trajectory. (I) Differential enrichment testing of the different treatment conditions across the gene expression modules along the pseudotime trajectory.

Code availability statement 
The R code underlying the single-cell RNA-seq analyses can be found in the associated Github repository under: https://github.com/rsankowski/car_cell_collaboration.
